src/xunit.runner.uap/MainPage.xaml.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+namespace XUnit.Runner.Uap
+{
+    public sealed partial class MainPage : Page
+    {
+
+        public MainPage()
+        {
+            this.InitializeComponent();
+            log = new StringBuilder();
+        }
+
+        internal static StringBuilder log;
+
+        public async void UpdateTextBox(string text)
+        {
+            await CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(
+                Windows.UI.Core.CoreDispatcherPriority.Normal,
+            () =>
+            {
+                log.AppendLine(text);
```

Comment:
I think the perf hit is worth it since that way we can retrieve those logs and see them in Helix.